Fix for missing includes, when compiling with GCC 7.3.0.

Henrik R. Nagel, 20190912
diff -ru ParaView-v5.5.2.orig/Plugins/SLACTools/pqSLACDataLoadManager.cxx ParaView-v5.5.2/Plugins/SLACTools/pqSLACDataLoadManager.cxx
--- ParaView-v5.5.2.orig/Plugins/SLACTools/pqSLACDataLoadManager.cxx	2018-06-15 21:09:45.000000000 +0200
+++ ParaView-v5.5.2/Plugins/SLACTools/pqSLACDataLoadManager.cxx	2019-08-23 12:25:29.000000000 +0200
@@ -36,6 +36,7 @@
 
 #include <QPushButton>
 #include <QtDebug>
+#include <QAction>
 
 class pqSLACDataLoadManager::pqUI : public Ui::pqSLACDataLoadManager
 {
diff -ru ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqColorMapEditor.cxx ParaView-v5.5.2/Qt/ApplicationComponents/pqColorMapEditor.cxx
--- ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqColorMapEditor.cxx	2018-06-15 21:09:45.000000000 +0200
+++ ParaView-v5.5.2/Qt/ApplicationComponents/pqColorMapEditor.cxx	2019-08-23 10:31:08.000000000 +0200
@@ -60,6 +60,7 @@
 #include <QKeyEvent>
 #include <QPointer>
 #include <QVBoxLayout>
+#include <QStyle>
 
 class pqColorMapEditor::pqInternals
 {
diff -ru ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqDoubleRangeSliderPropertyWidget.cxx ParaView-v5.5.2/Qt/ApplicationComponents/pqDoubleRangeSliderPropertyWidget.cxx
--- ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqDoubleRangeSliderPropertyWidget.cxx	2018-06-15 21:09:45.000000000 +0200
+++ ParaView-v5.5.2/Qt/ApplicationComponents/pqDoubleRangeSliderPropertyWidget.cxx	2019-08-23 11:19:44.000000000 +0200
@@ -43,6 +43,7 @@
 #include "vtkSMUncheckedPropertyHelper.h"
 
 #include <QGridLayout>
+#include <QStyle>
 
 class pqDoubleRangeSliderPropertyWidget::pqInternals
 {
diff -ru ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqStandardViewFrameActionsImplementation.cxx ParaView-v5.5.2/Qt/ApplicationComponents/pqStandardViewFrameActionsImplementation.cxx
--- ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqStandardViewFrameActionsImplementation.cxx	2018-06-15 21:09:45.000000000 +0200
+++ ParaView-v5.5.2/Qt/ApplicationComponents/pqStandardViewFrameActionsImplementation.cxx	2019-08-23 11:26:13.000000000 +0200
@@ -69,6 +69,7 @@
 #include <QPushButton>
 #include <QSet>
 #include <QShortcut>
+#include <QStyle>
 
 //-----------------------------------------------------------------------------
 pqStandardViewFrameActionsImplementation::pqStandardViewFrameActionsImplementation(
diff -ru ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqTimeInspectorWidget.cxx ParaView-v5.5.2/Qt/ApplicationComponents/pqTimeInspectorWidget.cxx
--- ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqTimeInspectorWidget.cxx	2018-06-15 21:09:45.000000000 +0200
+++ ParaView-v5.5.2/Qt/ApplicationComponents/pqTimeInspectorWidget.cxx	2019-08-23 11:31:06.000000000 +0200
@@ -53,6 +53,7 @@
 #include <QPainter>
 #include <QVariant>
 #include <QtDebug>
+#include <QHeaderView>
 
 namespace
 {
diff -ru ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqTransferFunctionWidgetPropertyDialog.cxx ParaView-v5.5.2/Qt/ApplicationComponents/pqTransferFunctionWidgetPropertyDialog.cxx
--- ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqTransferFunctionWidgetPropertyDialog.cxx	2018-06-15 21:09:45.000000000 +0200
+++ ParaView-v5.5.2/Qt/ApplicationComponents/pqTransferFunctionWidgetPropertyDialog.cxx	2019-08-23 11:37:36.000000000 +0200
@@ -35,6 +35,7 @@
 #include "pqTransferFunctionWidgetPropertyWidget.h"
 #include "vtkPiecewiseFunction.h"
 #include <QString>
+#include <QDoubleValidator>
 
 class pqTransferFunctionWidgetPropertyDialog::pqInternals
 {
diff -ru ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqViewResolutionPropertyWidget.cxx ParaView-v5.5.2/Qt/ApplicationComponents/pqViewResolutionPropertyWidget.cxx
--- ParaView-v5.5.2.orig/Qt/ApplicationComponents/pqViewResolutionPropertyWidget.cxx	2018-06-15 21:09:45.000000000 +0200
+++ ParaView-v5.5.2/Qt/ApplicationComponents/pqViewResolutionPropertyWidget.cxx	2019-08-23 12:02:12.000000000 +0200
@@ -39,6 +39,7 @@
 #include "vtkSMProxy.h"
 
 #include <QIntValidator>
+#include <QStyle>
 
 class pqViewResolutionPropertyWidget::pqInternals
 {
diff -ru ParaView-v5.5.2.orig/Qt/Components/pqChangeInputDialog.cxx ParaView-v5.5.2/Qt/Components/pqChangeInputDialog.cxx
--- ParaView-v5.5.2.orig/Qt/Components/pqChangeInputDialog.cxx	2018-06-15 21:09:45.000000000 +0200
+++ ParaView-v5.5.2/Qt/Components/pqChangeInputDialog.cxx	2019-08-22 08:47:35.000000000 +0200
@@ -48,6 +48,7 @@
 #include <QItemSelectionModel>
 #include <QLineEdit>
 #include <QRadioButton>
+#include <QHeaderView>
 
 class pqChangeInputDialog::pqInternals : public Ui::pqChangeInputDialog
 {
diff -ru ParaView-v5.5.2.orig/Qt/Components/pqCinemaTrackSelection.cxx ParaView-v5.5.2/Qt/Components/pqCinemaTrackSelection.cxx
--- ParaView-v5.5.2.orig/Qt/Components/pqCinemaTrackSelection.cxx	2018-06-15 21:09:45.000000000 +0200
+++ ParaView-v5.5.2/Qt/Components/pqCinemaTrackSelection.cxx	2019-08-23 09:44:43.000000000 +0200
@@ -50,6 +50,7 @@
 #include "pqTreeWidgetCheckHelper.h"
 #include "pqTreeWidgetItem.h"
 #include "ui_pqCinemaTrackSelection.h"
+#include <QHeaderView>
 
 // ============================================================================
 /// @brief Concrete implementation for seleciton of proxy arrays.
diff -ru ParaView-v5.5.2.orig/Qt/Python/pqPythonShell.cxx ParaView-v5.5.2/Qt/Python/pqPythonShell.cxx
--- ParaView-v5.5.2.orig/Qt/Python/pqPythonShell.cxx	2018-06-15 21:09:45.000000000 +0200
+++ ParaView-v5.5.2/Qt/Python/pqPythonShell.cxx	2019-08-23 08:51:46.000000000 +0200
@@ -52,6 +52,7 @@
 #include "vtkStringOutputWindow.h"
 #include "vtkWeakPointer.h"
 
+#include <QAbstractItemView>
 #include <QApplication>
 #include <QCursor>
 #include <QFile>
